koku/masu/external/downloader/ocp/ocp_report_downloader.py

```diff
@@ -143,7 +143,8 @@ def process_cr(report_meta):
             case_info = cr_status.get(case, {})
             if case_info.get("error"):
                 errors[case + "_error"] = case_info.get("error")
-        manifest_info["operator_errors"] = errors
+        if errors != {}:
+            manifest_info["operator_errors"] = errors
```

Comment:
just to prevent any future silly `keyErrors` could we do:
```suggestion
        manifest_info["operator_errors"] = errors or None
```

```python
>>> d1 = {}
>>> d2 = {'k1':'v1'}
>>> d2['errors'] = d1 or None
>>> print(d2)
{'k1': 'v1', 'errors': None}
```